app/models/language.rb

```diff
@@ -0,0 +1,3 @@
+class Language < ApplicationRecord
+  belongs_to :curriculum
```

Comment:
Should we make the proficiency field an enum?

Reply:
Done, used the LinkedIn levels of proficiency. https://www.linkedin.com/pulse/language-proficiency-linkedin-duaa-ali/